torchok/models/heads/segmentation/ocr.py

```diff
@@ -129,98 +130,37 @@ def forward(self, feats: Tensor, proxy_feats: Tensor) -> Tensor:
 
 
 @HEADS.register_class
-class HRNetSegmentationHead(BaseModel):
-    """HRNet segmentation head."""
-
-    def __init__(self, in_channels: int, num_classes: int):
-        """Init HRNetSegmentationHead.
-        Args:
-            in_channels: Size of each input sample.
-            num_classes: Number of classes.
-        """
-        super().__init__(in_channels, out_channels=num_classes)
-        self.num_classes = num_classes
-        self.convbnact = ConvBnRelu(in_channels,
-                                    in_channels,
-                                    kernel_size=1,
-                                    padding=0,
-                                    stride=1)
-        self.final_conv_layer = nn.Conv2d(in_channels=in_channels,
-                                          out_channels=num_classes,
-                                          kernel_size=1,
-                                          stride=1,
-                                          padding=0)
-
-    def forward(self, feats: Tensor) -> Tensor:
-        input_image, feats = feats
-        x = self.convbnact(feats)
-        out = self.final_conv_layer(x)
-        out = resize(out, input_image.shape[2:])
-        return out
-
-
-@HEADS.register_class
-class HRNetOCRSegmentationHead(BaseModel):
+class OCRSegmentationHead(SegmentationHead):
```

Comment:
It is better to write onw self.classifier inside class, rather than inherit from SegmentationHead